src/main/java/org/zapply/product/global/BaseTimeEntity.java

```diff
@@ -0,0 +1,28 @@
+package org.zapply.product.global;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.EntityListeners;
+import jakarta.persistence.MappedSuperclass;
+import lombok.Getter;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import java.time.LocalDateTime;
+
+@Getter
+@MappedSuperclass
+@EntityListeners(AuditingEntityListener.class)
+public abstract class BaseTimeEntity {
+
+    @CreatedDate
+    @Column(nullable = false, columnDefinition = "TIMESTAMP")
+    private LocalDateTime createdAt;
+
+    @LastModifiedDate
+    @Column(nullable = false, columnDefinition = "TIMESTAMP")
+    private LocalDateTime modifiedAt;
+
+    @Column(columnDefinition = "TIMESTAMP")
+    private LocalDateTime deletedAt;
```

Comment:
단순 soft-delete의 목적이라면, boolean isDeleted는 어떤가요?
```suggestion
    private Boolean isDeleted;
```

Reply:
저는 삭제 시점은 저장해두는게 좋다고 생각하기도 했고, 생성, 수정, 삭제 시점을 하나의 엔티티로 묶으면 생명주기를 모든 엔티티에 대해서 일관성 있게 관리할 수 있을 것 같아서 사용하고 있어요!